.travis.yml

```diff
@@ -1,18 +1,26 @@
 dist: jammy
 language: node_js
 node_js:
-    - node
+    - 21
 sudo: false
 install:
     - git clone --branch=develop --depth 1 https://github.com/enactjs/cli ../cli
     - pushd ../cli
     - npm install
     - npm link
     - popd
+    - git clone --branch=develop --depth 1 https://github.com/enactjs/enact ../enact
+    - pushd ../enact
+    - npm install
+    - npm run lerna exec -- --ignore enact-sampler --concurrency 1 -- npm --no-package-lock install
+    - npm run lerna exec -- --ignore enact-sampler --concurrency 1 -- npm --no-package-lock link
+    - npm run interlink
+    - popd
```

Comment:
could you please leave a comment under Additional Considerations why is this needed?